vulnerabilities/data_source.py

```diff
@@ -89,25 +115,43 @@ class Advisory:
     summary: str
     vulnerability_id: Optional[str] = None
     affected_packages: List[AffectedPackage] = dataclasses.field(default_factory=list)
+    fixed_packages: List[AffectedPackage] = dataclasses.field(default_factory=list)
```

Comment:
We discussed the OSV design and fixed should be under affected_packages

Reply:
Done!